src/views/Export/AssessmentExportSelection/index.tsx

```diff
@@ -146,36 +156,46 @@ function AssessmentExportSelection(props: OwnProps) {
     return (
         <div className={_cs(className, styles.exportSelection)}>
             <div className={styles.leftContainer}>
-                <Cloak
-                    {...viewsAcl.arys}
-                    render={
-                        <PrimaryButton
-                            className={styles.button}
-                            onClick={handleAssessmentExportClick}
-                            disabled={exportPending}
-                            pending={
-                                exportPending && exportClass === EXPORT_CLASS.assessmentExport
-                            }
-                        >
-                            {_ts('export', 'startAssessmentExportButtonLabel')}
-                        </PrimaryButton>
-                    }
-                />
-                <Cloak
-                    {...viewsAcl.arys}
-                    render={
```

Comment:
Why remove the cloak?

Reply:
The cloak is shifted to the whole page instead of the button